### DISCUSSION TITLE
Show game or title in live notifications toasts

### DISCUSSION THREAD

**Discussion by xHeaveny** [Chatterino/chatterino2#3205] [Ideas] [answered]
Show the title/ game in the live notifications 

Thans in the moment the notification 
![image](https://user-images.githubusercontent.com/69117321/130612602-5a5108ec-336f-4bd0-97a2-c5f09ca0073c.png)

After that I should look like that 
![Screenshot 2021-08-24 135958](https://user-images.githubusercontent.com/69117321/130613255-a7965488-13c8-499a-8b01-2c9364823af4.png)

**Comment by ALazyMeme** [comment]
Duplicate of https://github.com/Chatterino/chatterino2/issues/1297
But closing https://github.com/Chatterino/chatterino2/issues/1297 in favour of this issue

**Comment by apa420** [comment]
Post enhancements in discussions just as the issue templates suggest
![](https://i.nuuls.com/KaKU-.png)

**Comment by 8thony** [comment, marked as answer]
- Implemented since #2044
